app/controllers/puppetca_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
     # expire cache if forced
     Rails.cache.delete("ca_#{@proxy.id}") if params[:expire_cache] == "true"
     certs = find_certs
-    @certificates = certs.sort.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @certificates = certs.sort
```

Comment:
The previous behavior was that they arrive sorted (sort from the server) and the can be re-sorted on the client